README.md

```diff
@@ -8,6 +8,15 @@ The [Cromwell documentation has a dedicated site](https://cromwell.readthedocs.i
 
 First time to Cromwell? Get started with [Tutorials](https://cromwell.readthedocs.io/en/stable/tutorials/FiveMinuteIntro/).
 
+### Google LifeSciences is being deprecated in July - migrate to GCP Batch
```

Comment:
I don't know if "deprecated" is strong enough here - we should make it clear that it will actually stop working.